0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -164,6 +166,13 @@ The sender uses the receiver details to create the ILP packet:
 
 Note that the sender can send as many payments as they want using the same receiver info. The sender SHOULD query the receiver again once the time indicated in the [`Cache-Control` header](#response-headers) has passed.
 
+### Payment Result
+
+When the receiver fulfills a payment that was set up through spsp, they may add a signature of the OER-encoded fulfillment data packet using the `signature_info` provided.
+Likewise, when the receiver rejects a payment that was set up through spsp, they may sign the OER-encoded ILP error using the `signature_info`. This will allow the sender,
+as well as (if they know the receiver's public key) intermediate connectors, to know that this failure or success was not a fake result that was intended to game some other
+connector's reliability statistics.
```

Comment:
What if an intermediate connector has to issue an error? From where would the other parties involved in the payment obtain his pubkey?

Reply:
That would stay unsigned.